src/crimson/os/seastore/omap_manager/btree/btree_omap_manager.cc

```diff
@@ -37,60 +37,61 @@ BtreeOMapManager::initialize_omap(Transaction &t)
 }
 
 BtreeOMapManager::get_root_ret
-BtreeOMapManager::get_omap_root(omap_root_t &omap_root, Transaction &t)
+BtreeOMapManager::get_omap_root(const omap_root_t &omap_root, Transaction &t)
 {
   assert(omap_root.omap_root_laddr != L_ADDR_NULL);
   laddr_t laddr = omap_root.omap_root_laddr;
-  return omap_load_extent(get_omap_context(omap_root, t), laddr, omap_root.depth);
+  return omap_load_extent(get_omap_context(t), laddr, omap_root.depth);
 }
 
 BtreeOMapManager::handle_root_split_ret
-BtreeOMapManager::handle_root_split(omap_context_t oc, OMapNode::mutation_result_t mresult)
+BtreeOMapManager::handle_root_split(omap_root_t &omap_root, omap_context_t oc,
```

Comment:
Why did we break omap_root out of omap_context_t?

Reply:
omap root only updated in btree_omap_manager.  and omap_context_t used in omap_btree_node_impl.cc

Reply:
Not really sure I get it, but it's fine for now.